lib/fauna/query.rb

```diff
@@ -91,7 +105,8 @@ def var(name)
     #
     # Reference: {FaunaDB Basic Forms}[https://faunadb.com/documentation/queries#basic_forms]
     def if_(condition, then_, else_)
-      { if: condition, then: then_, else: else_ }
+      # rubocop:disable Style/HashSyntax
```

Comment:
using 1.9 syntax caused a syntax error without the braces. I disabled the rubcop rule here because it was warning. It seems like a good general rule, however.